### PR TITLE
Optimize p2p

### PR DESCRIPTION
**Pull request by lochjin**

### CODE REVIEW

**Review by dindinw — COMMENTED** [review]
Please use go1.3 std error and remove the dependency from `pkg/errors`
see https://blog.golang.org/go1.13-errors for details.

**Comment by lochjin** [comment]
> Please use go1.3 std error and remove the dependency from `pkg/errors`
> see https://blog.golang.org/go1.13-errors for details.

https://github.com/Qitmeer/qitmeer/pull/409/commits/b54e97acc2ce8f28a07d9e0c6eca629b12d1660d

**Review by dindinw — CHANGES_REQUESTED** [review]
Please fix the build  `Errorf call has arguments but no formatting directives`,  BTW,  you might also execute a `gofmt` additionally  
```golang	
	return nil, fmt.Errorf("could not get multiaddr:w", err)
```

File: cmd/crawler/node/node.go [hunk 1]
```diff
@@ -413,7 +413,7 @@ func convertToSingleMultiAddr(node *qnode.Node) (multiaddr.Multiaddr, error) {
 	multiAddrString := fmt.Sprintf("/ip4/%s/tcp/%d/p2p/%s", ip4.String(), node.TCP(), id)
 	multiAddr, err := multiaddr.NewMultiaddr(multiAddrString)
 	if err != nil {
-		return nil, fmt.Errorf("could not get multiaddr:w", err)
+		return nil, fmt.Errorf("could not get multiaddr:w%", err)
```
Comment (dindinw) [inline cmd/crawler/node/node.go:416]: typo here, should be `%w`

**Review by dindinw — APPROVED** [review]

**Pull request merged**